src/averell/utils.py

```diff
@@ -284,3 +284,40 @@ def read_features(corpus_folder):
 
     features = sorted(features, key=lambda i: i['poem_title'])
     return features
+
+
+def pretty_string(text, n_words):
+    """Add a line break every number of words into a text to create multiline
+    cells to use in `tabulate_corpora`
+
+    :param text: String to be converted
+    :param n_words: Number of words in which the line break will be added
```

Comment:
` Number of words in which the line break will be added`
Esto significa "número de palabras tras las que añadir un salto de línea"? Si es así, quizás mejor "Number of words to add a line break after"?

Reply:
Done